ISISComputingGroup/EPICS-ioc
# Ticket1403 multiple eurotherms update

### Description of work

Ensure that IOC works for 6 Eurotherms groups each of which can have 7 Eurotherms connected to it.

### To test

https://github.com/ISISComputingGroup/IBEX/issues/1403

Crash is fixed via std new release which is already on master

See pull requests:

1. https://github.com/ISISComputingGroup/ibex_gui/pull/296
1. https://github.com/ISISComputingGroup/EPICS-ReadASCII/pull/1

### Acceptance criteria

Multiple Eurotherms can be started with different lesser and greater addresses set. These are shown in the GUI.

#### Code Review

- [ ] Is there an [emulator](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Emulating-Devices) for the device? **No emulator I have not changed the IOC that much**

### Functional Tests


### Final steps
